modules/storage-gcs/gcs/gcs.go

```diff
@@ -244,6 +244,11 @@ func (g *gcs) DestinationPath(className, snapshotID string) string {
 		makeObjectName(className, snapshotID, "snapshot.json"))
 }
 
+func (g *gcs) InitSnapshot(ctx context.Context, className, snapshotID string) (*snapshots.Snapshot, error) {
+	// TODO: implement
```

Comment:
This should be done in a separate PR, yes?

Reply:
Yes correct :)